docs/kafka/getting-started-kafka/quickstart.yml

```
@@ -4,6 +4,9 @@ metadata:
   name: getting-started
   annotations:
     order: 1
+  labels:
+    serviceCategory: Application Services
```

From K8s doc:
```
Valid label value:

must be 63 characters or less (can be empty),
unless empty, must begin and end with an alphanumeric character ([a-z0-9A-Z]),
could contain dashes (-), underscores (_), dots (.), and alphanumerics between.
```
You might want to update the value of the `serviceCategory` to `application-services`.
Also, it's not very common to use camelcase for the key of the labels, so you want also change that to `service-category`.
```suggestion
    service-category: application-services
```